modules/gridmap/doc_classes/GridMap.xml

```diff
@@ -96,6 +96,20 @@
 				This function returns always the map set on the GridMap node and not the map on the NavigationServer. If the map is changed directly with the NavigationServer API the GridMap node will not be aware of the map change.
 			</description>
 		</method>
+		<method name="get_octant_coords_from_cell_coords" qualifiers="const">
+			<return type="Vector3i" />
+			<param index="0" name="cell_coords" type="Vector3i" />
+			<description>
+				Returns the [Vector3i] octant coordinates of the octant that the cell with cell coordinates [param cell_coords] belongs to.
+			</description>
+		</method>
+		<method name="get_octants_in_bounds" qualifiers="const">
+			<return type="Vector3i[]" />
+			<param index="0" name="bounds" type="AABB" />
+			<description>
+				Returns an array of [Vector3i] octant coordinates that fall inside the [param bounds], including octants that have no cells in use.
```

Comment:
```suggestion
				Returns an array of [Vector3i] octant coordinates that are inside the given [param bounds], including octants that have no cells in use.
```
Is it necessary to specify "**[Vector3i]** octant coordinates" every time?

Reply:
Yeah, else the universe could fall apart. More serious, what is the concern / harm?

Reply:
It just doesn't feel right to use it as a shorter form of "__Vector3i representing the octant coordinates_". Plus the return type can be inferred from the function's signature and it's not particularly unusual.